server/pkg/item/field_change.go

```diff
@@ -0,0 +1,85 @@
+package item
+
+
+type FieldChangeType string
+
+const (
+	Add    FieldChangeType = "add"
+	Update FieldChangeType = "update"
+	Delete FieldChangeType = "delete"
+)
+
+
+type FieldChange struct {
+	Id *FieldID
+	CurrentValue any
+	PreviousValue any
```

Comment:
```suggestion
	CurrentValue *value.Multiple
	PreviousValue *value.Multiple
```

Reply:
@rot1024 , by the way this one already has `any` type in `internal/adapter/gql/gqlmodel/convert_value.go`

```
func ToValue(v *value.Multiple, multiple bool) any {
	if !multiple {
		return v.First().Interface()
	}
	return v.Interface()
}
```

Changed it to:

```
	CurrentValue []interface{}
	PreviousValue []interface{}
```
as it is an interface of that value.

Reply:
This struct cannot be used as is in the GraphQL world.In the domain model layer, the domain model information should not be stripped away unnecessarily. Instead of using it in the GraphQL layer, process it as a domain model in the use case layer and convert it to GraphQL.